google/cloud/pubsublite/internal/wire/subscriber_impl.py

```diff
@@ -111,7 +111,7 @@ def _handle_response(self, response: SubscribeResponse):
             )
             return
         # Workaround for incredibly slow proto-plus-python accesses
-        messages = list(response.messages.messages._pb)
+        messages = list(response.messages.messages._pb) # pytype: disable=attribute-error
```

Comment:
Need to determine the cause of `attribute-error` here

Reply:
No idea- but proto-plus is so dynamic it doesn't surprise me. This is accessing the internal _pb member to avoid proto plus. I don't know why there are so many unrelated changes included with this PR though?

Reply:
@dpcollins-google, I added manual some changes to clean up owlbot.py and add consistency with other python googleapis/python-* repositories . If you'd like to regenerate the PR without any manual changes click the checkbox `Regenerate this pull request now` in the PR body. Please also see the comment from @PicardParis here: https://github.com/googleapis/python-pubsublite/pull/388#issuecomment-1337004201